Objects/exceptions.c

```diff
@@ -1063,6 +1080,54 @@ BaseExceptionGroup_str(PyObject *op)
         self->msg, num_excs, num_excs > 1 ? "s" : "");
 }
 
+static PyObject *
+BaseExceptionGroup_repr(PyObject *op)
+{
+    PyBaseExceptionGroupObject *self = PyBaseExceptionGroupObject_CAST(op);
+    assert(self->msg);
+
+    PyObject *exceptions_str = NULL;
+
+    /* If the initial exceptions string was not saved in the constructor. */
```

Comment:
```suggestion
    /* Use the initial exceptions string, if any. */
```

and swap the if-else branches. Or just remove this comment, I do not think it adds much value.